src/index.js

```diff
@@ -0,0 +1,93 @@
+import './styles.css';
+import getMenu from './modules/getmenu.js';
+import logo from './assets/resources/logo.png';
+import mealicon from './assets/resources/mealicon.png';
+import mealcountericon from './assets/resources/iconcounter.png';
+import { clickLoveBtn, getNumberOfLikes } from './modules/likes.js';
+import itemCounter from './modules/itemscounter.js';
+
+const parser = new DOMParser();
+
+const logoImage = new Image();
+logoImage.src = logo;
+logoImage.className = 'logo-img';
+logoImage.alt = 'Logo Image';
+const logoDiv = document.querySelector('.logo');
+logoDiv.append(logoImage);
+
+const logoIcon = new Image();
+logoIcon.src = mealicon;
+logoIcon.className = 'logo-icon-homepage';
+logoIcon.alt = 'Logo Icon in the Homepage';
+const logoIconDiv = document.querySelector('.logo-icon-home');
+logoIconDiv.append(logoIcon);
+
+const logoIcon2 = new Image();
+logoIcon2.src = mealicon;
+logoIcon2.className = 'logo-icon-homepage';
+logoIcon2.alt = 'Logo Icon in the Homepage';
+const logoIconDiv2 = document.querySelector('.logo-icon-home2');
+logoIconDiv2.append(logoIcon2);
+
+// let menuArray = [];
+window.onload = async () => {
+  const menuArray = await getMenu();
+  const likesArray = await getNumberOfLikes();
+  const likedMenuArray = [];
+
+  for (let i = 0; i < menuArray.length; i += 1) {
```

Comment:
- [Optional] Nice working on your menu, you can use es6 [map](https://stackoverflow.com/questions/56117235/use-array-map-on-an-2-dimensional-array) method here instead of a for loop for cleaner code.